src/main/resources/templates/build.gradle.template

```diff
@@ -46,10 +48,26 @@ task downloadAndUnzipFile(dependsOn: downloadBaseFilesystemZip, type: Copy) {
   into file(srcInterlokDir)
 }
 
+def chmodExec(path) {
+  project.exec {
+    commandLine("chmod", "+x", path)
+  }
+}
+
 // Do we need that or the previous step is enough?
 task downloadAndUnzipFileAndCopyConfigDir(dependsOn: downloadAndUnzipFile, type: Copy) {
   from file(srcInterlokDir + "config/")
   into interlokTmpConfigDirectory
+  
+  doLast {
+    if (!DefaultNativePlatform.getCurrentOperatingSystem().isWindows()) {
+      from fileTree(srcInterlokDir) {
+        include "bin/start-interlok"
+      }.each {
+        chmodExec(it.path)
+      }
+    }
+  }
```

Comment:
Is it worth just doing this (might not be syntatically correct)

```xml
doLast {
  ant.chmod(file: "$srcInterlokDir/bin/start-interlok", perm: "+x", failonerror: false)
}
```
Since ant already does the detection of unix under the covers (c.f. http://ant.apache.org/manual/Tasks/chmod.html).  This still expects "chmod" to be on the path, just like what you're doing, so we still run the risk of someone not having a fully POSIX compliant machine - wouldn't surprise me in the "corporate managed environment")

We might consider using http://ant.apache.org/manual/Tasks/setpermissions.html as a platform independent task as well?  which means something like (provided the gradle bundles ant >= 1.10)

```
ant.setpermissions(mode="755") {
  file(file: "$srcInterlokDir/bin/start-interlok")
}
```

Reply:
Also, the buildParent should point to the v4 variant in adaptris/interlok-build-parent

Reply:
Switching to ant.chmod "works on the mac" but should still be tested on windows etc.